content/millennium/r4/financial/charge-item.md

```diff
@@ -86,10 +86,15 @@ Search for ChargeItems that meet supplied query parameters:
  Name                  | Required?  | Type          | Description
 -----------------------|------------|---------------|-------------------------------------------------------
  `_id`                 | Yes        | [`token`]     | The logical resource id associated with the ChargeItem.
+ `context`             | Yes        | [`token`]     | Encounter associated with event.
+[`_count`]             | No         | [`number`]    | The maximum number of results to return.
+  `-status`            | No         | [`token`]     | The status of the encounter. Example: `billable`
 
 Notes:
 
 - When searching by `_id`, only a single ChargeItem id can be provided.
+- `_count` and `-status` must be provided with context.
+- `-status` valid parameter is `billable`
```

Comment:
let's reword this to say something like:
`-status` must be set to `billable`

Reply:
+1 to rewording

Reply:
updated here https://github.com/cerner/fhir.cerner.com/pull/665/commits/aa37a5294afd32516a75bd83c1abd309654fb0e9#diff-16d4f8f74626c62d57a8b71d6be699f60e72fffed8636bc32e6654d4252b3ebbR97

Reply:
nit: can we add a period to the end of it to match the other notes

Reply:
updated here https://github.com/cerner/fhir.cerner.com/pull/665/commits/6782742b24d6e089283790fdc2d3280dc576d991#diff-16d4f8f74626c62d57a8b71d6be699f60e72fffed8636bc32e6654d4252b3ebbR97